pkgs/applications/audio/muse/default.nix

```diff
@@ -38,7 +73,9 @@ stdenv.mkDerivation rec {
       MusE aims to be a complete multitrack virtual studio for Linux,
       it is published under the GNU General Public License.
     '';
-    license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ orivej ];
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [ eclairevoyant orivej ];
+    platforms = [ "x86_64-linux" ];
```

Comment:
I see mentions of RPi in the issues, maybe `lib.platforms.linux`-would be a better fit? CI will test-build on `aarch64-linux` for you, and I can verify on my ARM hardware.